hackathon_site/event/tests.py

```diff
@@ -22,3 +24,10 @@ def test_creates_team_if_not_provided(self):
         profile = Profile.objects.create(user=self.user, status="Accepted")
         self.assertEqual(Team.objects.count(), 1)
         self.assertEqual(Team.objects.first(), profile.team)
+
+
+class IndexViewTestCase(TestCase):
```

Comment:
I think it'd be good to have test cases for HTTP_200 regardless of if you're logged in or not (aka add a test case for logged in please)

Reply:
As we discussed externally, there's no way that being logged in could effect this view. So while technically for completeness it wouldn't hurt to have such a test, that opens us up to over-testing for things that can never happen.